dynamic/abbreviation_test.go

```diff
@@ -0,0 +1,51 @@
+package dynamic
+
+import (
+	"fmt"
+	"testing"
+)
+
+func TestAbbreviation(t *testing.T) {
+	aStrings := []string{
```

Comment:
Convert the three different slices to a single struct slice:
```go
tests := []struct{
  a, b string
  expected bool
}{}
```

Reply:
Done!